Resources/Maps/saltern.yml

```diff
@@ -40729,4 +40729,10 @@ entities:
   - pos: -36.5,4.5
     parent: 853
     type: Transform
+- uid: 4724
+  type: SpawnPointCentcomOfficial
+  components:
+  - pos: -36.5,-6.5
+    parent: 853
+    type: Transform
```

Comment:
This shouldn't really be part of the map unless you remove the ghost role component from it